keras_cv/layers/patch_encoder.py

```diff
@@ -0,0 +1,21 @@
+
+import tensorflow as tf
+from tensorflow.keras import layers
+
+"""
+Linear projection of patches made with `Patching` layer
+"""
+#@tf.keras.utils.register_keras_serializable(package="keras_cv")
+class PatchEncoder(layers.Layer):
+    def __init__(self, num_patches, projection_dim):
```

Comment:
for Keras layers its always good to take a **kwargs argument and propogate it to super().__init__()